UnityProject/Assets/Scripts/Objects/Engineering/APC.cs

```diff
@@ -448,7 +448,7 @@ private bool EmergencyState
 		public MultitoolConnectionType ConType => MultitoolConnectionType.APC;
 
 		[SerializeField]
-		private bool multiMaster = true;
+		private bool multiMaster = true; //TODO
```

Comment:
your todo is to move these at the top
not just this line, everything at the top as well, i saw properties hiding in the middle of functions

Reply:
yeah, multimaster need to be worked on

Reply:
that's not my point, my point is that there are variables in the middle of functions